File: script/c++/notes2.cpp
// Copyright (c) 2025 Joseph Chierichella | All rights reserved

#include <iostream>
#include <string>
#include <vector>

using namespace std;

vector<string> vecn;   // name of notes
vector<string> vecno;  // notes

int create() {
    string name;
    cout << "Enter name of the file (one word without '.') : ";
    cin >> name;
    vecn.push_back(name);
    string notes;
    cout << name << ".txt" << endl;
    cout << ": ";
    cin.ignore();
    getline(cin, notes);
    vecno.push_back(notes);
    return 0;
}

void display() {
    cout << "\n--- Notes List ---" << endl;
    for (size_t i = 0; i < vecn.size(); ++i) {
        cout << vecn[i] << ".txt: " << vecno[i] << endl;
    }
    cout << "------------------" << endl;
}

int main() {
    int UserInput;
    while (UserInput != 3) {
        cout << "       NOTEPAD" << endl;
        cout << "~~~~~~~~~~~~~~~~~~~~~" << endl;
        cout << " 1 ) MAKE NEW NOTES" << endl;
        cout << " 2 ) VIEW NOTES" << endl;
        cout << " 3 ) EXIT PROGRAM" << endl;
        int UserInput;
        cout << "!: ";
        cin >> UserInput;
        if (UserInput == 3) {
            cout << "EXITING PROGRAM" << endl;
            break;
        }
        else if (UserInput == 1) {
            create();
        }
        else if (UserInput == 2) {
            display();
        }
    }


    return 0;
}

File: script/python/notes2.py
# Copyright (c) 2025 Joseph Chierichella | All rights reserved

notes = ""

def nNotes():
    global notes
    print("Enter Notes\n")
    input2 = input(": ")
    if notes:
        notes += "\n"
    notes += input2

def vNotes():
    global notes
    print("     Notes")
    print("---------------")
    print(notes)    

i = 0
while i == 0:
    print("      Notes")
    print("---------------")
    print(" 1 ) New notes")
    print(" 2 ) View notes")
    print(" 3 ) Quit program")
    input1 = int(input("!: "))
    if input1 == 3:
        print("Quiting program!")
        i = 1
        break
    elif input1 == 1:
        nNotes()
    elif input1 == 2:
        vNotes()
    else:
        print("SYNTAX ERROR")
